pkg/cli/flags.go

```diff
@@ -164,6 +158,11 @@ func init() {
 			flag.Hidden = true
 		} else if flag.Name == logflags.ShowLogsName {
 			flag.Hidden = true
+		} else if flag.Name == logflags.LogToStderrName {
+			// The default value is overriden in cli.Main. We don't override it here
+			// as doing so would affect all of the cli tests and any package which
+			// depends on cli.
```

Comment:
I find this comment confusing. We are overriding the default here.

Reply:
I've expanded the comment. Is it clearer now?